chain/vm/runtime.go

```diff
@@ -142,7 +142,7 @@ func (rt *Runtime) StorePut(x cbor.Marshaler) cid.Cid {
 
 var _ rt0.Runtime = (*Runtime)(nil)
 var _ rt5.Runtime = (*Runtime)(nil)
-var _ rt6.Runtime = (*Runtime)(nil)
```

Comment:
nit why do you need to keep around assert on rt5 but not rt6?  Can you remove all but rt7? If there's a breaking change to the runtime interface, same method name different parameter types this pattern won't work.

Reply:
Based on a quick offline sync, we think it's best to just to have the assertion for every actors version.

If the breaking change @ZenGround0 describes above does happen, we will have a broader refactor to make here.